packages/@tinacms/sharedctx/src/edit-state-ctx.tsx

```diff
@@ -79,7 +80,9 @@ export const EditProvider: React.FC = ({ children }) => {
 export const useEditState = () => {
   const { edit, setEdit } = useContext(EditContext)
   if (!setEdit) {
-    throw new Error('No `TinaEditProvider` found')
+    throw new Error(
+      'Unable to find `TinaProvider`; did you forget to add the TinaCMS container to your _app layout?  See our setup docs: https://tina.io/docs/introduction/tina-init/#adding-tina'
```

Comment:
Only nitpick is that `_app` is a Next.js-specific thing, we may forget to amend this later on.

Reply:
Talked with @jeffsee55 about this and we agreed to replace `_app` with `app root` to remove the Next.js-specific language 👍